public/app/plugins/datasource/influxdb/types.ts

```diff
@@ -27,8 +27,8 @@ export interface InfluxSecureJsonData {
 
 export interface InfluxQueryPart {
   type: string;
-  params?: string[];
-  interval?: string;
+  params?: Array<string | number>;
+  interval?: string; // FIXME: is this ever used?
```

Comment:
What's the plan for all FIXMEs? Is it something you're planning to look at in this PR?

Reply:
they are things that should be improved at some point, but they are above the scope of this pull-request. for example, the mentioned `interval` here, it is probably not used anymore, and can be removed, but doing so creates extra work in this pull-request, maybe unit tests need to be adjusted etc. the idea is that later one can come back and clean these up.